roles/ipabackup/library/ipabackup_get_backup_dir.py

```diff
@@ -55,18 +55,20 @@
   type: str
 '''
 
-from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ansible_ipa_backup import IPAAnsibleModule
 from ipaplatform.paths import paths
 
 
 def main():
-    module = AnsibleModule(
+    module = IPAAnsibleModule(
         argument_spec=dict(),
         supports_check_mode=True,
     )
 
-    module.exit_json(changed=False,
-                     backup_dir=paths.IPA_BACKUP_DIR)
+    module.exit_raw_json(
```

Comment:
Is the use of exit_raw_json really needed for this module? There are no input parameters

Reply:
fixed.